anesthetic/weighted_pandas.py

```diff
@@ -72,13 +75,25 @@ def wrapper(*args, **kwargs):
         wrapper.__name__ = name
         return wrapper
```

Comment:
I believe that with `_op_via_apply` we no longer need _make_wrapper?` (it's not [covered](https://app.codecov.io/gh/handley-lab/anesthetic/commit/b68bb6f33a61450f25558300ca1e2095efd16cb2/blob/anesthetic/weighted_pandas.py#L64) at the moment)